dev_guide/jobs.adoc

```diff
@@ -56,14 +56,32 @@ spec:
       - name: pi
         image: perl
         command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-      restartPolicy: Never
+      restartPolicy: OnFailure    <4>
 ----
 
 1. Optional value for how many pod replicas a job should run in parallel; defaults to `*completions*`.
 2. Optional value for how many successful pod completions are needed to mark a job completed; defaults to one.
 3. Template for the pod the controller creates.
+4. The restart policy of the pod. This does not apply to the job controller. See xref:creating-a-job-known-issues[] for details.
 ====
 
+Alternatively, you can create and launch a job from a single command -- specifically, `oc run`. The following command creates and launches the same job as the `pi` resource:
+
+----
+$ oc run pi --image=perl --replicas=1  --restart=OnFailure --command -- perl -Mbignum=bpi -wle 'print bpi(2000)' 
+----
+
+[[creating-a-job-known-issues]]
+=== Known Limitations
+
+The job specification restart policy only applies to _pods_, not the _job controller_. However, the job controller is hard-coded to keep retrying jobs to completion. 
+ 
+As such, `restartPolicy: Never` or `--restart=Never` will effectively result in the same behavior as `restartPolicy: OnFailure` or `--restart=OnFailure`. That is, when a job fails it will be restarted until it succeeds (or is manually discarded). The policy only sets which subsystem performs the restart.
+ 
+With `Never`, the _job controller_ performs the restart. With each attempt, the job controller will increment the number of failures in the job status and create new pods. Unfortunately, this means that with each failed attempt, the number of pods increases. 
+ 
+With `OnFailure`, _kubelet_ performs the restart. Each attempt will not increment the number of failures in the job status. In addition, kubelet will retry failed jobs starting pods on the same nodes.
```

Comment:
Fixed in latest commit.

@gaurav-nelson: Slight edit -- instead of including the parameter key, I just refer to it as _policy_, as in:

> With the `Never` policy, the _job controller_ performs ...
...
> With the `OnFailure` policy, _kubelet_ ...

The policy is fed differently if you're using a spec VS launching from command (**restartPolicy: Never** VS **--restart=Never** ) but I want to avoid quoting both methods, for brevity. It's the parameter value that matters anyway, not the key.